kube/services/revproxy/00nginx-config.yaml

```diff
@@ -7,7 +7,7 @@ kind: ConfigMap
 metadata:
   name: revproxy-nginx-conf
 data:
-  userid.js: |
+  helpers.js: |
```

Comment:
This is great - really love this approach.
Let's go ahead and pull this out to its own file `helpers.js`, and create a new configmap in `kube-setup-revproxy`:
   ```
gen3 update_config revproxy_helper_js .../helpers.js
  ```
, and also add a `helperSpec.js` that runs `helper.js` through some test cases - I like `jasminejs` for that kind of thing:
```
const helpers = require('./helpers.js');

describe('the revproxy helpers module', function() {
  it('can decode a JWT token', function() { ... });
  it('does MathAbs', function() { ... }
...
});
```
 - then add this to the `Jenkinfile`:
```
npx jasmine .../helpersSpec.js
```